documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -422,7 +422,7 @@ Let's look at what a _create_/_read_ event value might look like for our `custom
 
 If we look at the `schema` portion of this event's _value_, we can see the schema for the _envelope_ is specific to the collection, and the schema for the `source` structure (which is specific to the MongoDB connector and reused across all events). Also note that the `after` value is always a string, and that by convention it will contain a JSON representation of the document.
 
-If we look at the `payload` portion of this event's _value_, we can see the information in the event, namely that it is describing that the document was read as part of an snapshot (since `op=r` and `initsync=true`), and that the `after` field value contains the JSON string representation of the document.
+If we look at the `payload` portion of this event's _value_, we can see the information in the event, namely that it is describing that the document was read as part of an snapshot (since `op=r` and `snapshot=true`), and that the `after` field value contains the JSON string representation of the document.
```

Comment:
This is still named init sync in source.

Reply:
Are you sure? We left it like that in offsets, but not in `sourceInfo`. Also see the example above, it has `"snapshot" : false`.